src/map/lua/lua_baseentity.cpp

```diff
@@ -10285,7 +10285,7 @@ inline int32 CLuaBaseEntity::updateClaim(lua_State *L)
 inline int32 CLuaBaseEntity::addStatusEffect(lua_State *L)
 {
     TPZ_DEBUG_BREAK_IF(m_PBaseEntity == nullptr);
-    TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype == TYPE_NPC);
+    //TPZ_DEBUG_BREAK_IF(m_PBaseEntity->objtype == TYPE_NPC);
```

Comment:
Would be good to uncomment/update any break_if's where the conditions have changed

Reply:
This was a boo boo made by me testing early on, fixed now ✔